orderbook/models/orderbook_types_manual.go

```diff
@@ -1,34 +1,113 @@
 package models
 
-import "time"
+import (
+	"math/big"
+	"time"
+)
```

Comment:
Lets keep structs and methods in the same package, let's put all the models into orderbook pkg

Reply:
The advantage of keeping these in a sub-folder is we can maintain a cleaner viewing space for people looking at our code when using the API. I expand on this further in the other comment.

Reply:
It does not make any sense to keep it in different packages, it breaks the common sense and does not make anything cleaner